### PR TITLE
Replace OnRack with RackHD

### PR DESCRIPTION
**Pull request by christineday**

### CODE REVIEW

**Comment by christineday** [comment]
there is still a reference to renisar in this topic.  What should it be replaced with?

**Comment by bossidy** [comment]
There are similar entries that probably need suppressing. For example, the section on getting the distribution media has:

curl http://xfer.renasar.com/emc/ubuntu_trusty.tgz | tar xvzf -
curl http://xfer.renasar.com/emc/esxi.tgz | tar xvzf -

File: docs/monorail/tasks.rst [hunk 1]
```diff
@@ -171,7 +171,7 @@ job constructor) and run that job.
 **Defining a Job**
 
 To create a job, define a subclass of
-[Job.Base](https://<server>:<port>/projects/ONRACK/repos/on-tasks/browse/lib/jobs/base-job.js)
+[Job.Base](https://<server>:<port>/projects/RackHD/repos/on-tasks/browse/lib/jobs/base-job.js)
```
Comment (heckj) [inline docs/monorail/tasks.rst]: This link was meant to be a link to the source code in source control - so the relevant link here is:

    https://github.com/RackHD/on-tasks/blob/master/lib/jobs/base-job.js

File: docs/monorail/tasks.rst [hunk 2]
```diff
@@ -239,7 +239,7 @@ At render time, values are rendered if the exist in the task render context. The
 - **options**
     - This refers to the task definition options itself. Mainly for referencing values in substrings that will eventually be defined by a user (e.g. `'sudo mv {{ options.targetFile }} /tmp/{{ options.targetfile }}'` )
 - **context**
-    - This refers to the shared context object that all tasks in a graph have R/W access to. Enables one task to use values produced by another at runtime. For example, the [ami catalog provider task](https://<server>:<port>/projects/ONRACK/repos/on-tasks/browse/lib/task-data/tasks/provide-catalog-ami-bios-version.js) gets the most recent catalog entry for the AMI bios, whose value can be referenced by other tasks via `{{ context.ami.systemRomId }}`
+    - This refers to the shared context object that all tasks in a graph have R/W access to. Enables one task to use values produced by another at runtime. For example, the [ami catalog provider task](https://<server>:<port>/projects/RackHD/repos/on-tasks/browse/lib/task-data/tasks/provide-catalog-ami-bios-version.js) gets the most recent catalog entry for the AMI bios, whose value can be referenced by other tasks via `{{ context.ami.systemRomId }}`
```
Comment (heckj) [inline docs/monorail/tasks.rst]: correct source code link here is:

    https://github.com/RackHD/on-tasks/blob/master/lib/task-data/tasks/provide-catalog-ami-bios-version.js

**Comment by heckj** [comment]
We'll want to come up with different instructions on how to get the source media for the ubuntu repositories and ESXi where relevant an explain how to make an install those files instead of referencing the non-existing renasar transfer files we made for easy access earlier

**Pull request merged**